frontend/pages/RegistrationPage/Breadcrumbs/_styles.scss

```diff
@@ -97,7 +96,7 @@
 
       &.registration-breadcrumbs__page--complete {
         &::before {
-          background-color: $core-white;
+          background: $core-white;
```

Comment:
main bug fix: `background-color` was not overriding background color as it was set with other attributes using `background` so needed to be set using `background` as well